package.json

```diff
@@ -69,13 +69,10 @@
     "is-ci": "^3.0.1",
     "jest": "^29.3.1",
     "jest-environment-jsdom": "^29.4.2",
-    "path-browserify": "^1.0.1",
     "prettier": "^2.8.3",
     "readline": "^1.3.0",
     "sass": "^1.56.1",
     "sass-loader": "^13.2.0",
-    "stream-browserify": "^3.0.0",
```

Comment:
Since we dont need this package `stream-browserify` do we still need it in the `webpack.common.js`?
https://github.com/CryptKeeperZK/crypt-keeper-extension/blob/60713467b351f728894048f83a4803d813131a42/webpack.common.js#L90